.github/workflows/fendermint-docker-build.yaml

```
@@ -0,0 +1,90 @@
+name: Fendermint Build Docker
```

The image has both fendermint and ipc in it. Let's drop Fendermint from everywhere, it's just the IPC image.

```suggestion
name: Build Docker Image
```